README.md

```diff
@@ -9,6 +9,28 @@
 
 A flexible front end to plate bases pipelines in Sequencescape
 
+## Initial Setup
+
+1. In Sequencescape, configure all Limber required data:
+```shell
+  bundle exec rake limber:setup
+```
+
+1. Start Sequencescape (will start in 3000):
+```shell
+  bundle exec rails s
+```
```

Comment:
If it worth mentioning the delayed job as well?

Reply:
Yes, I'll add it